konvoy: Update docs for konvoy release v1.5.0-beta.5 (#2963)
mesosphere/dcos-docs-site: pages/ksphere/konvoy/1.5.0-beta/install/install-secured/index.md

On highly secured clusters you may need to modify the `cluster.yaml` file with additional options.
See the sample file below for possible changes that may be applied in your cluster.

## Kubernetes CVE Patches

> At times, CVEs may be discovered in the Kubernetes codebase. Based on the severity and the impact of a specific CVE, you may want to temporarily use alternative docker images for the core Kubernetes components instead of the default `k8s.gcr.io` repository.

Reviewer: ```suggestion
At times, CVEs can be discovered in the Kubernetes codebase. Based on the severity and impact of a specific CVE, you can temporarily use alternative docker images for the core Kubernetes components instead of the default `k8s.gcr.io` repository.
```
